### ISSUE TITLE
The One Probe not working

### ISSUE THREAD

**Issue by manthoszh** [McJty/TheOneProbe#511]
Describe the bug
The One Probe mod is not showing extra information. Oddly enough it was not always like this. It worked as expected for a day or two then went into this lessened state. The in game config is set to extended and none of the mods setting were changed. I also feel like the "(woodT0)" did not use to be there as well.

This is on a private server running the All the Mods 6: To the sky mod pack. I used the curseforge server pack to get it up and running for a friend and myself.

We ran into a ticking block entity crash and had to restart the world from a backup but The One Probe mod problem existed before the crash. Just an FYI don't know if it matters.

To Reproduce
Nothing need be done for us. This is just how it is when we log on

Screenshots
![2021-09-02_13 05 54](https://user-images.githubusercontent.com/71455308/132055971-77b79a04-beb9-41b7-91f2-83259b0ca706.png)
Chest with contents not sneaking

![2021-09-02_13 05 59](https://user-images.githubusercontent.com/71455308/132055988-de0b6f17-1ead-4d8f-9697-52060b70fc2f.png)
Chest with contents while sneaking

![2021-09-02_13 06 03](https://user-images.githubusercontent.com/71455308/132056006-6d7a1f63-c222-4aac-9de8-188229fd5673.png)
Proof of contents

![2021-09-02_13 06 17](https://user-images.githubusercontent.com/71455308/132056034-da3a3468-91e3-4deb-b930-a27d8e951df9.png)
Won't show extra information with looking at Fired Crucible, should show fluid volume and a heating level.

![2021-09-02_13 06 41](https://user-images.githubusercontent.com/71455308/132056048-ecec6478-44f5-4e72-ab38-ce07c1cbbbee.png)
Shows a little extra content

**Comment by McJty** [comment]
Version please

**Comment by manthoszh** [comment]
theoneprobe-1.16-3.1.4.jar
All the Mods 6 - To the Sky - ATM6sky - Skyblock-1.1.8.zip

**Comment by McJty** [comment]
Most likely the config got changed. Try the /top config or /top need commands and see if they are still correct

**Comment by manthoszh** [comment]
Yes they are set to extended view. I also tried all of the other options to see if anything would change and it didn't. 'Needed' took away the box as expected but 'Not Needed' and 'Extended' showed the same info.

Below are the configs on the computer I play from, not the server:
[theoneprobe-common.txt](https://github.com/McJtyMods/TheOneProbe/files/7110317/theoneprobe-common.txt)
[theoneprobe-client.txt](https://github.com/McJtyMods/TheOneProbe/files/7110318/theoneprobe-client.txt)

If you want to look at the servers configs let me know.

**Comment by tytan652** [comment]
It seems The One Probe get reset each time that a mod is updated, many complains about this issue on the Ragnamod VI modpack issue tracker in french.

At each update the modpacker need to remember to re-change the TOP config each time.

**Comment by TonioGela** [comment]
+1 on this I got it too

**Comment by Trobyne** [comment]
> Most likely the config got changed. Try the /top config or /top need commands and see if they are still correct



> Version please

Tis is also happening to me on the same version. Please help

**Comment by McJty** [comment]
I can't do anything with this. I really need more information. Exact versions. Exact description of what you're doing

**Comment by Trobyne** [comment]
The issue is that the progress bar or info is not showing up. Just the information of the objest looking at and how to break it
![this](https://user-images.githubusercontent.com/109169710/178566630-c88acdc1-f525-4830-91ac-d97ae239c566.jpg)
 
The file name is "theoneprobe-1.16-3.1.7"
I have done nothing specificity only playing a modpack called "Seaopolis"
After going through playing the modpack i have use the piece of paper that you can change the option to show information to show "extended". Nothing changed in the tooltip box. I changed the and it still did not show the extended information. I went through the config file and change the setting As the progress to be "show as text". Nothing changed.

**Comment by McJty** [comment]
What about other blocks? Extended info on chests? Or blocks from other mods?

**Comment by Trobyne** [comment]
Even looking at the Tinkers construct basin it only shows the equivalent of the above picture. With "the one probe" paper it seems to only change whether the tool tip box is on or off.

**Comment by McJty** [comment]
What's the value of this in theoneprobe-common config:
#Is the probe needed to show the tooltip? 0 = no, 1 = yes, 2 = yes and clients cannot override, 3 = probe needed for extended info only
#Range: 0 ~ 3
needsProbe = 3

I saw it is 3 in the original reporters config. It should be 0

**Comment by Trobyne** [comment]
I successfully changed that config option and it works 
The RF/FE power bar shows up for the "block breaker" machine but no progress bar for anything cooling in tinkers construct basin

**Comment by Trobyne** [comment]
Just to give you an example I went on another modpack to get this picture 
![20220712_202922](https://user-images.githubusercontent.com/109169710/178578408-59e90447-3831-4fbb-b267-60d40b83414f.jpg)
Would it be easier to coppy the config files over from this modpack to the current one I am playing to get the progress bar up for other object?
I don't think it is possible as the two modpacks use different versions of TOP
